lib/config/migrations/custom/package-rules-migration.ts

```diff
@@ -25,6 +26,29 @@ function renameKeys(packageRule: PackageRule): PackageRule {
   return newPackageRule;
 }
 
+function removeMatchLanguage(packageRule: PackageRule): PackageRule[] {
+  const newPackageRules: PackageRule[] = [];
+  const matchLanguages = packageRule.matchLanguages;
+  // no migration needed
```

Comment:
```suggestion
  delete packageRule.matchLanguages;
  // no migration needed
```
we should always delete the old key (can be null or an empty array.